package.json

```diff
@@ -34,18 +37,39 @@
 	"devDependencies": {
 		"enzyme": "^3.3.0",
 		"enzyme-adapter-react-16": "^1.1.1",
+		"eslint-plugin-react-native": "^3.2.1",
 		"husky": "^1.0.0-rc.10",
 		"lint-staged": "^7.2.0",
 		"prettier": "^1.13.7",
 		"puppeteer": "^1.5.0"
 	},
 	"eslintConfig": {
+		"parser": "babel-eslint",
+		"extends": [
+			"eslint:recommended",
+			"plugin:react/recommended"
+		],
 		"parserOptions": {
 			"ecmaVersion": 9,
 			"sourceType": "module",
 			"ecmaFeatures": {
 				"jsx": true
 			}
+		},
+		"globals": {
```

Comment:
Sorry for not mentioning this before, we can set the environments we're using to avoid specifying these manually, e.g.

```
"eslintConfig": {
  "envs": {
    "node": true,
    "browser": true,
    "jest": true
  }
}
```